baseline/OSPS-AC.yaml

```
@@ -54,7 +54,12 @@ controls:
       - reference-id: SAMM
         identifiers: 
           - Operations -Environment Management -Configuration Hardening Lvl1
-    assessment-requirements:
+      - reference-id: PCIDSS
+        identifiers: 
+          - 2.2.1
+          - 8.2.1
+          - 8.3.1
+     assessment-requirements:
```

I think this one has an extra space:

```suggestion
    assessment-requirements:
```

---

it did!